android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -436,6 +482,25 @@ public void finishNativeInitialization() {
         if (RateUtils.getInstance(this).shouldShowRateDialog())
             showBraveRateDialog();
 
+        int visitedNewsCardsCount = SharedPreferencesManager.getInstance().readInt(
+                BravePreferenceKeys.BRAVE_NEWS_CARDS_VISITED);
+
+        // for Brave Newsa initialize the no. of cards to 0
+        SharedPreferencesManager.getInstance().writeInt(
+                BravePreferenceKeys.BRAVE_NEWS_CARDS_VISITED, 0);
```

Comment:
`BRAVE_NEWS_CARDS_VISITED`, `BRAVE_NEWS_CARDS_VIEWED`, ... - are per-session counters?

Reply:
yes, session as in a new tab is a new session too. but i will remove them from BraveActivity. they are not used there anymore.

Reply:
seems that this part was removed, marking as resolved